src/zigbee-migration-selection.html

```diff
@@ -13,19 +13,19 @@
   <div class="layout horizontal justified guides">
     <a href="/migrate-zigbee-zha-only">
       <div class="layout vertical center">
-        <img src="/images/network-migrate.png"" width="250"/>
+        <img src="/images/icons/zha-logo.png" width="250"/>
```

Comment:
Instead of including the logos directly, you can refer to our brands repo. Those also include dark mode icons https://github.com/home-assistant/brands/tree/master/core_integrations/deconz

Reply:
Thank you for the hint @balloob ! 
When I reference the icons from the brand repo, the text underneath does not align vertically. I tried to fix that by using inline CSS to force the text to align. Then I read somewhere that using inline CSS like that is bad style. 
Therefore, I chose to create new icons that have all the same height. 
Not sure which would be the better solution...

Reply:
To solve that, wrap each icon in a div with a fixed height.

Reply:
Done. Thank you for the support!